### ISSUE TITLE
Can I use spritejs for gamedev?

### ISSUE THREAD

**Issue by luizbills** [spritejs/spritejs#11]

**Comment by akira-cn** [comment]
Yes, you can. I think SpriteJS is quite suitable for 2D gamedev. 
It can easily work with matter-js or proton.

Here some example http://spritejs.org/demo
